tools/launch_benchmark_suite_scheduler_slurm.py

```diff
@@ -0,0 +1,82 @@
+import sys
+
+import submitit
+from fvcore.common.file_io import PathManager
+from vissl.config.attr_dict import AttrDict
+from vissl.engines.benchmark_suite_scheduler import BenchmarkSuiteScheduler
+from vissl.utils.io import load_file
+
+
+# Default slurm options to pass to the executor.
+_DEFAULT_SLURM_OPTIONS = {
+    "NAME": "vissl",
+    "COMMENT": "vissl evaluation job",
+    "PARTITION": "learnlab",
+    "CONSTRAINT": "",
+    "TIMEOUT_MIN": 72 * 60,  # Timeout in minutes.
+    "CPUS_PER_TASK": 8,
+    "MEM_GB": 16,
+    "ADDITIONAL_PARAMETERS": {},
+}
+
+
+class SlurmEvaluatorJob:
```

Comment:
nit: add a small docstring for the class. important for documentation purposes in vissl sphinx docs